internal/testutils/authctl.go

```diff
@@ -17,6 +17,7 @@ func BuildAuthctl() (binaryPath string, cleanup func(), err error) {
 	binaryPath = filepath.Join(tempDir, "authctl")
 
 	cmd := exec.Command("go", "build", "-o", binaryPath, "./cmd/authctl/main.go")
+	cmd.Args = append(cmd.Args, GoBuildFlags()...)
```

Comment:
Not sure if this still applies, but @denisonbarbosa code (I think he did the initial use of it), explicitly stated that the argument is positional and needs to go just after `run|build`, so maybe this should be changed (and also made it clear in the function docstring.

Reply:
right. I'll make sure that we do `cmd.Args = append(cmd.Args, GoBuildFlags()...)` comes before appending any other arguments.